backend/tests/test_auto_fail.py

```diff
@@ -132,6 +132,41 @@ def now(cls, *arg: list) -> datetime:
     return mydatetime
 
 
+@fixture
+def admin_token(client: TestClient) -> str:
+    """Get an admin token for authentication"""
+    response = client.post(
+        "/login",
+        data={
+            "username": os.environ.get("ADMIN_USERNAME", ""),
+            "password": os.environ.get("ADMIN_PASSWORD", ""),
+        },
+    )
+    token = response.json()["access_token"]
+    return token
+
+
+@fixture
+def workspace_api_key(client: TestClient, admin_token: str) -> str:
```

Comment:
Should we move these to conftest so it's available to all the tests?

Reply:
Moved, thanks :)